src/plugin/file/zip/ui/zipprocessstep.js

```diff
@@ -0,0 +1,200 @@
+
+
+/**
+ * ZIP import file selection step
+ *
+ * @extends {os.ui.wiz.step.AbstractWizardStep.<plugin.file.zip.ZIPParserConfig>}
+ * @constructor
+ */
+plugin.file.zip.ui.ZIPProcessStep = function() {
+  plugin.file.zip.ui.ZIPProcessStep.base(this, 'constructor');
+  this.template = '<zipprocessstep></zipprocessstep>';
+  this.title = 'Unzip file(s)';
+};
+
+
+goog.inherits(plugin.file.zip.ui.ZIPProcessStep, os.ui.wiz.step.AbstractWizardStep);
+
+
+/**
+ * @inheritDoc
+ */
+plugin.file.zip.ui.ZIPProcessStep.prototype.finalize = function(config) { };
+
+
+/**
+ * The ZIP import file selection step directive
+ *
+ * @return {angular.Directive}
+ */
+plugin.file.zip.ui.processStepDirective = function() {
+  return {
+    restrict: 'E',
+    replace: true,
+    templateUrl: os.ROOT + 'views/plugin/zip/zipprocessstep.html',
+    controller: plugin.file.zip.ui.ZIPProcessStepCtrl,
+    controllerAs: 'processStep'
+  };
+};
+
+
+/**
+ * Add the directive to the module
+ */
+os.ui.Module.directive('zipprocessstep', [plugin.file.zip.ui.processStepDirective]);
+
+
+
+/**
+ * Controller for the ZIP import file selection step
+ *
+ * @param {!angular.Scope} $scope
+ * @param {!angular.JQLite} $element
+ * @param {!angular.$timeout} $timeout
+ * @constructor
+ * @ngInject
+ */
+plugin.file.zip.ui.ZIPProcessStepCtrl = function($scope, $element, $timeout) {
+  /**
+   * @type {?angular.Scope}
+   * @private
+   */
+  this.scope_ = $scope;
+
+  /**
+   * @type {?angular.$timeout}
+   * @private
+   */
+  this.timeout_ = $timeout;
+
+  /**
+   * @type {plugin.file.zip.ZIPParserConfig}
+   * @private
+   */
+  this.config_ = /** @type {plugin.file.zip.ZIPParserConfig} */ ($scope['config']);
+
+  /**
+   * @type {boolean}
+   */
+  this['loading'] = false;
+
+  /**
+   * @type {boolean}
+   */
+  this['valid'] = false;
+
+  /**
+   * @type {Array.<String>}
+   */
+  this['msg'] = [];
+
+  /**
+   * @type {number}
+   */
+  this['wait'] = 0;
+
+  this.scope_.$watch('config["status"]', this.onStatusChange.bind(this), false);
+  this.scope_.$watch('config["files"]', this.onFilesChange.bind(this), false);
+
+  $scope.$on('$destroy', this.destroy_.bind(this));
+
+  this.validate_();
+
+  goog.log.info(plugin.file.zip.ui.ZIPProcessStepCtrl.LOGGER_, 'SUCCESS!  Initialized zip -- Process Step');
+};
+
+
+/**
+ * Logger
+ * @type {goog.log.Logger}
+ * @private
+ * @const
+ */
+plugin.file.zip.ui.ZIPProcessStepCtrl.LOGGER_ = goog.log.getLogger('plugin.file.zip.ui.ZIPProcessStepCtrl');
+
+
+/**
+ * @param {Array.<Object>} val
+ * @param {Array.<Object>} oldVal
+ * @param {Object} scope
+ * @public
+ */
+plugin.file.zip.ui.ZIPProcessStepCtrl.prototype.onFilesChange = function(val, oldVal, scope) {
+//  this.validate_();
+};
+
+
+/**
+ * @param {number} val
+ * @param {number} oldVal
+ * @param {Object} scope
+ * @public
+ */
+plugin.file.zip.ui.ZIPProcessStepCtrl.prototype.onStatusChange = function(val, oldVal, scope) {
+//  this.validate_();
```

Comment:
Are these meant to be commented out?

Reply:
Ahh, those are old code that's no longer used.  I'm removing this entire step, so those will be gone soon too.